### ISSUE TITLE
Add library charges for monatomic ions?

### ISSUE THREAD

**Issue by davidlmobley** [openforcefield/openforcefields#5]
I've just run into an issue where I get an opaque charging failure if i try to parameterize a system containing sodium and chloride ions. I think this is because the toolkits don't now how to charge these ions. Now that we have support for library charges, we should likely just include charges for simple monoatomic ions (at least those for which our FFs have LJ parameters) in the file.

Does this seem reasonable to you @j-wags and @SimonBoothroyd ? (Tagging Simon since I know he'd been using these.)

Error follows:

```
-------------------------------------------------------------------
Exception                         Traceback (most recent call last)
<ipython-input-41-5d652a00e67a> in <module>
     20 # This is also true for ions
     21 #system = ff.create_openmm_system(off_topology, charge_from_molecules = mols)
---> 22 system = ff.create_openmm_system(off_topology, charge_from_molecules = [host_OFFmol])
     23 
     24 # Set up system

~/anaconda3/envs/drugcomp/lib/python3.7/site-packages/openforcefield/typing/engines/smirnoff/forcefield.py in create_openmm_system(self, topology, **kwargs)
   1136         # Add forces and parameters to the System
   1137         for parameter_handler in parameter_handlers:
-> 1138             parameter_handler.create_force(system, topology, **kwargs)
   1139 
   1140         # Let force Handlers do postprocessing

~/anaconda3/envs/drugcomp/lib/python3.7/site-packages/openforcefield/typing/engines/smirnoff/parameters.py in create_force(self, system, topology, **kwargs)
   2937             toolkit_registry = kwargs.get('toolkit_registry', GLOBAL_TOOLKIT_REGISTRY)
   2938             temp_mol.generate_conformers(n_conformers=10, toolkit_registry=toolkit_registry)
-> 2939             temp_mol.compute_partial_charges_am1bcc(toolkit_registry=toolkit_registry)
   2940 
   2941             # Assign charges to relevant atoms

~/anaconda3/envs/drugcomp/lib/python3.7/site-packages/openforcefield/topology/molecule.py in compute_partial_charges_am1bcc(self, toolkit_registry)
   2012             charges = toolkit_registry.call(
   2013                       'compute_partial_charges_am1bcc',
-> 2014                       self
   2015             )
   2016         elif isinstance(toolkit_registry, ToolkitWrapper):

~/anaconda3/envs/drugcomp/lib/python3.7/site-packages/openforcefield/utils/toolkits.py in call(self, method_name, *args, **kwargs)
   3023                 method = getattr(toolkit, method_name)
   3024                 try:
-> 3025                     return method(*args, **kwargs)
   3026                 except NotImplementedError:
   3027                     pass

~/anaconda3/envs/drugcomp/lib/python3.7/site-packages/openforcefield/utils/toolkits.py in compute_partial_charges_am1bcc(self, molecule)
   1271 
   1272         if quacpac_status is False:
-> 1273             raise Exception('Unable to assign charges')
   1274 
   1275         # Extract and return charges

Exception: Unable to assign charges
```

**Comment by SimonBoothroyd** [comment]
Related to https://github.com/openforcefield/openforcefield/issues/305

I agree we should add these to the force fields.

**Comment by dotsdl** [comment]
Taking this one on as part of my work on #305. I believe these should be solved in the same PR.

**Comment by j-wags** [comment]
Ah, sorry to be late to the party. I agree that we should add these to our force fields, and that using the integer values is unambiguously the right thing to do.

**Comment by dotsdl** [comment]
How would we like to proceed with pulling in the additions from openforcefield/openforcefield#305?

**Comment by j-wags** [comment]
Per some discussion with @SimonBoothroyd @dotsdl and @mattwthompson earlier today, I think that we should make two NEW force fields immediately (corresponding to FOUR new files) -- 
* `openff-1.0.1.offxml`: 1.0.0 w/ ion charges
* `openff_unconstrained-1.0.1.offxml`: 1.0.0 unconstrained w/ ion charges
* `openff-1.1.1.offxml`: 1.1.0 w/ ion charges
* `openff_unconstrained-1.1.0-.offxml`: 1.1.0 unconstrained w/ ion charges

We should also plan for `1.2.0`, our next release, to have ion charges included in it. 

This numbering scheme may be a little confusing for people, but the alternatives (don't change the version, or add a `-ions` suffix, both seem to be worse). And, after all, this IS why we included a bugfix number in the versions.

**Comment by davidlmobley** [comment]
100% agree. Thanks.

**Comment by dotsdl** [comment]
This has been addressed in #9 and #10. Releases have been cut corresponding to each, and packages will be built by tomorrow.
